editor/code_editor.h

```diff
@@ -174,6 +174,8 @@ class CodeTextEditor : public VBoxContainer {
 
 	Label *info = nullptr;
 	Timer *idle = nullptr;
+	float idle_time = 0.0f;
+	float idle_time_with_errors = 0.0f;
```

Comment:
These should probably have sensible defaults, and/or we need to make sure to read the editor settings early enough, as in current `master` this PR causes:
```
ERROR: Time should be greater than zero.
   at: set_wait_time (./scene/main/timer.cpp:83)
```